fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -109,9 +110,9 @@ class OffloadConfig:
     """Class for specifying all arguments related to offloading parameters."""
 
     # Offload type: currently only supports: "ssd_offload"
-    offload_type: str = None
+    offload_type: Optional[str] = None
     # Path to the directory for storing parameters offloaded to disk.
-    ssd_filepath_dir: str = None
+    dir: Optional[str] = None
```

Comment:
This is a breaking change for existing users. Should we raise an exception if we see `ssd_filepath_dir` or maybe mention this in the Changelog entry?

Reply:
I will note this in the changelog.